app/controllers/conferences_controller.rb

```diff
@@ -33,8 +32,15 @@ def redirect
 
   private
 
-  def generate_gid(team)
-    "#{Season.current.name}-#{Time.now.getutc.to_i}-#{team.id}"
+  def build_conference
+    conference = Conference.new(conference_params)
+    conference.season_id = current_season.id
+    conference.gid = generate_gid
+    conference
+  end
+
+  def generate_gid
+    "#{Season.current.name}-#{Time.now.getutc.to_i}-#{current_user.id}"
```

Comment:
The button to add new conference are supposed to appear just for a student of a specific team and orga members. However, orga members do not have a team, so I thought that use the current user id in the GID could help to track who added the conference, instead what team added the conference. But as far as I know, just the students (and orga members) are allowed to add conferences. You are right, I have to include tests for it! :D thank you